reference/conanfile/methods.rst

```diff
@@ -23,13 +23,13 @@ control. But if the source code is available in a repository, you can directly g
         settings = "os", "compiler", "build_type", "arch"
 
         def source(self):
-            self.run("git clone https://github.com/memsharded/hello.git")
+            self.run("git clone https://github.com/conan-io/hello.git")
             # You can also change branch, commit or whatever
-            # self.run("cd hello && git checkout 2fe5...")
+            # self.run("cd hello")
```

Comment:
Remove the previous comment or revert the `git checkout ...`

Reply:
You are right. Thanks